edge-util/src/Microsoft.Azure.Devices.Edge.Util/concurrency/AsyncLock.cs

```diff
@@ -8,7 +8,6 @@ namespace Microsoft.Azure.Devices.Edge.Util.Concurrency
 
     public sealed class AsyncLock : IDisposable
```

Comment:
We should probably move on to using AsyncLock in Nito.AsyncEx. Doesn't have to be in this PR though.